content/nginx-one/agent/metrics/configure-otel-metrics.md

```diff
@@ -39,3 +39,196 @@ You can validate that metrics are successfully exported by using the methods bel
+      enabled: true
+
+service:
+  pipelines:
+    metrics/prometheus-example-pipeline:
+      receivers:
+        - nginxplus
+      processors:
+        - resource/default
+      exporters:
+        - prometheus
+```
+
+{{% /tab %}}
+{{% tab name="NGINX OSS" %}}
+
+```yaml
+exporters:
+  prometheus:
+    endpoint: "127.0.0.1:5643"
+    resource_to_telemetry_conversion:
+      enabled: true
+
+service:
+  pipelines:
+    metrics/prometheus-example-pipeline:
+      receivers:
+        - nginx
+      processors:
+        - resource/default
+      exporters:
+        - prometheus
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+- **Third-party OTel Collector**
+
+{{< tabs name="third-party-collector" >}}
+{{% tab name="NGINX Plus" %}}
+
+```yaml
+exporters:
+  otlp/local-collector:
+    endpoint: "my-local-collector.com:443"
+    timeout: 10s
+    retry_on_failure:
+      enabled: true
+      initial_interval: 10s
+      max_interval: 60s
+      max_elapsed_time: 10m
+    tls:
+      insecure: true
+
+service:
+  pipelines:
+    metrics/otlp-example-pipeline:
+      receivers:
+        - nginxplus
+      processors:
+        - resource/default
+      exporters:
+        - otpl/local-collector
+```
+
+{{% /tab %}}
+{{% tab name="NGINX OSS" %}}
+
+```yaml
+exporters:
+  otlp/local-collector:
+    endpoint: "my-local-collector.com:443"
+    timeout: 10s
+    retry_on_failure:
+      enabled: true
+      initial_interval: 10s
+      max_interval: 60s
+      max_elapsed_time: 10m
+    tls:
+      insecure: true
+
+service:
+  pipelines:
+    metrics/otlp-example-pipeline:
+      receivers:
+        - nginx
+      processors:
+        - resource/default
+      exporters:
+        - otpl/local-collector
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+- **Add Debug Exporter**
+
+{{< tabs name="debug-exporter" >}}
+{{% tab name="NGINX Plus" %}}
+
+```yaml
+exporters:
+  debug:
+    verbosity: detailed
+    sampling_initial: 5
+    sampling_thereafter: 200
+
+service:
+  pipelines:
+    metrics/debug-example-pipeline:
+      receivers:
+        - nginxplus
+      processors:
+        - resource/default
+      exporters:
+        - debug
+```
+
+{{% /tab %}}
+{{% tab name="NGINX OSS" %}}
+
+```yaml
+exporters:
+  debug:
+    verbosity: detailed
+    sampling_initial: 5
+    sampling_thereafter: 200
+
+service:
+  pipelines:
+    metrics/debug-example-pipeline:
+      receivers:
+        - nginx
+      processors:
+        - resource/default
+      exporters:
+        - debug
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+
+### Troubleshooting
+
+To view the merged OpenTelemetry configuration set the Agent log level to debug in `/etc/nginx-agent/nginx-agent.conf` and restart NGINX Agent 
+
+1. Edit the configuration file `sudo vim /etc/nginx-agent/nginx-agent.conf`
+2. Change the log property
+```yaml
+log:
+  level: debug 
+```
+3. Restart NGINX Agent 
+4. View merged OpenTelemetry configuration `cat /var/lib/nginx-agent/opentelemetry-collector-agent-debug.yaml`
```

Comment:
``````suggestion
1. Edit the configuration file `sudo vim /etc/nginx-agent/nginx-agent.conf`
1. Change the log property
   ```yaml
   log:
     level: debug 
   ```
1. Restart NGINX Agent, run as root:
   ```shell
   systemctl restart nginx-agent
   ```
1. View the merged OpenTelemetry configuration `cat /var/lib/nginx-agent/opentelemetry-collector-agent-debug.yaml`
``````